vmdb/app/controllers/miq_request_controller.rb

```diff
@@ -83,15 +89,11 @@ def button
 
   def request_edit
     assert_privileges("miq_request_edit")
-    prov = MiqRequest.find_by_id(params[:id])
-    if prov.workflow_class
-      @org_controller = prov.resource_type == "MiqHostProvisionRequest" ? "host" : "vm"                           #request originated for resource_type
-      @redirect_controller = "miq_request"
-      @refresh_partial = "prov_edit"
-      @req_id = params[:id]
-      @prov_type = prov.resource_type
+    provision_request = MiqRequest.find_by_id(params[:id])
```

Comment:
Can we use the newer ActiveRecord syntax here:  MiqRequest.where(:id => params[:id]